react-responsive/react-responsive.d.ts

```diff
@@ -0,0 +1,66 @@
+// Type definitions for react-responsive 1.1.3
+// Project: https://github.com/contra/react-responsive
+// Definitions by: Alexey Svetliakov <https://github.com/asvetliakov>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="../react/react.d.ts" />
+
+declare module "react-responsive" {
+    import * as React from 'react';
+
+    namespace MediaQuery {
+        export interface MediaQueryProps {
+            query?: string;
+            // matchers
+            orientation?: "portrait" | "landscape";
+            scan?: "progressive" | "interlace";
+            aspectRatio?: string;
```

Comment:
`aspectRatio` is a `string`? The [css spec](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries#aspect-ratio) says it consists of two numbers